### PR TITLE
fixes and changes the recently introduced 'alignas' to be 'align'

### PR DESCRIPTION
**Pull request by Araq**

### CODE REVIEW

File: doc/manual.rst [hunk 1]
```diff
@@ -6433,38 +6433,39 @@ generates:
...
-always ignored.
+Align pragma
+------------
+
+The `align`:idx: pragma is for variables and object field members. It
+modifies the alignment requirement of the entity being declared. The
+argument must be a constant power of 2. Valid non-zero
+alignments that are weaker than nother align pragmas on the same
+declaration are ignored. Alignments that are weaker that the
+alignment requirement of the type are ignored.
```
Comment (krux02) [inline doc/manual.rst]: Why did you remove the mentioning of the `0` argument? `0` is not a power of two nor a valid alignment, but the behavior is well specified as ignored. There is logic for that and there is a test case for that.

Comment (Araq) [reply doc/manual.rst]: I know, I'll remove it, what's the point of it? Copying bad design is not good design.

Comment (krux02) [reply doc/manual.rst]: @cooldome @mratsim Since you reviewed my original PR, I would like to have your opinion about this.

**Comment by Araq** [comment]
> alternatively you can also provide a good reason why the 0 argument is a bad idea and this behavior should not be taken over from C.

How about you explain to me why 0 should be allowed? What's the point? "because C does it" is not a reason.

**Comment by Araq** [comment]
Unrelated CI failure, merging.

**Comment by krux02** [comment]
> How about you explain to me why 0 should be allowed? What's the point? "because C does it" is not a reason.

Why? What's the point of it? You don't want to hear it anyway. If you would have cared, you wouldn't have dismissed my review of this PR in the same moment of asking that question.

**Comment by Araq** [comment]
Fair enough I figured it out in the meantime and the feature is not required, you can easily use other solutions in Nim when the situation comes up.
